Makefile

```diff
@@ -9,12 +9,11 @@ default:
 .PHONY: test-all
 test-all: test
 	make test -C aws
-	make test -C hacking
+	tox
```

Comment:
Between the switch to `tox` and previous changes that were made, it appears several files are no longer needed:

* this `Makefile`
* all 3 `test-requirements.txt` files
* the empty `hacking/requirements.txt` file